### ISSUE TITLE
A few crashes during OECD team testing (10:15ish, 10:34AM and 10:49AM Paris time) 

### ISSUE THREAD

**Issue by OECDPILA** [pila-project/pila-platform#126]
(High Priority)

Screen goes white and then everyone has to refresh. It took about a minute to come back up. We were around 15 people on the system.

**Comment by jasonford** [comment]
Thanks for the times! I was able to narrow down the exact error, and it was the same in all 3 cases. I'll implement a fix and provide more details on the cause when I have them for sure

**Comment by jasonford** [comment]
I opened up an issue on our main platform repo for this: https://github.com/knowlearning/platform/issues/23

I implemented a patch on our side, referenced there, which is now live. I'll leave that issue open for tracking since a deeper fix probably needs an update to Deno (might open an issue there if I can generate a reproduction). The deeper fix isn't necessary to fix things with our current patch though.
